Task2/task2.py

```diff
@@ -0,0 +1,154 @@
+        time_out = '{} {} {}'.format(hours[date_today_hour],hour_text[0],no_minutes).capitalize()
+        print(time_out)
+    elif (date_today_hour > 1) and (date_today_hour < 5):
+        time_out = '{} {} {}'.format(hours[date_today_hour],hour_text[1],no_minutes).capitalize()
+        print(time_out)
+    elif (date_today_hour >= 5) and (date_today_hour <= 12):
+        time_out = '{} {} {}'.format(hours[date_today_hour],hour_text[2],no_minutes).capitalize()
+        print(time_out)
+    elif (date_today_hour == 13):
+        time_out = '{} {} {}'.format(hours[date_today_hour],hour_text[0],no_minutes).capitalize()
+        print(time_out)
+    elif (date_today_hour > 13) and (date_today_hour < 17):
+        time_out = '{} {} {}'.format(hours[date_today_hour],hour_text[1],no_minutes).capitalize()
+        print(time_out)
+    elif (date_today_hour >= 17) and (date_today_hour < 24):
+        time_out = '{} {} {}'.format(hours[date_today_hour],hour_text[2],no_minutes).capitalize()
+        print(time_out)
+    elif (date_today_hour == 00):
+        time_out = '{} {} {}'.format(hours[date_today_hour],hour_text[2],no_minutes).capitalize()
+        print(time_out)
+elif (date_today_minutes < 30) and (date_today_minutes > 0):
+    if (date_today_minutes == 21) or (date_today_minutes == 1):
+        time_out = '{} {} {}'.format(minutes[date_today_minutes],minutes_text[0],hours2[date_today_hour]).capitalize() 
+        print(time_out)   
+    elif ((date_today_minutes > 1) and (date_today_minutes < 5)) or ((date_today_minutes > 21) and (date_today_minutes < 25)):
+        time_out = '{} {} {}'.format(minutes[date_today_minutes],minutes_text[1],hours2[date_today_hour]).capitalize()
+        print(time_out)
+    else:
+        time_out = '{} {} {}'.format(minutes[date_today_minutes],minutes_text[2],hours2[date_today_hour]).capitalize()
+        print(time_out)
+elif (date_today_minutes == 30):
+    time_out = 'половина {}'.format(hours2[date_today_hour]).capitalize()
+    print(time_out)
+elif (date_today_minutes > 30) and (date_today_minutes < 45):
+    if (date_today_minutes == 31) or (date_today_minutes == 41):
+        time_out = '{} {} {}'.format(minutes[date_today_minutes],minutes_text[0],hours2[date_today_hour]).capitalize()
+        print(time_out)
+    elif ((date_today_minutes > 31) and (date_today_minutes < 35)) or ((date_today_minutes > 41) and (date_today_minutes < 45)):
+        time_out = '{} {} {}'.format(minutes[date_today_minutes],minutes_text[1],hours2[date_today_hour]).capitalize()
+        print(time_out)
+    elif (date_today_minutes > 34) and (date_today_minutes < 41):
+        time_out_atime_oututo = '{} {} {}'.format(minutes[date_today_minutes],minutes_text[2],hours2[date_today_hour]).capitalize()
+        print(time_out)
+elif (date_today_minutes >= 45) and (date_today_minutes < 60):
+    time_out = 'без {} {} {}'.format(minutes2[date_today_minutes],minutes_text[2],hours2[hour_inp]).capitalize()
+    print(time_out)
```

Comment:
this is totally redundant part, it's a repeat of the main logic to process the system time, but instead you could just ask user, what they want to see here, a custom time or a system, and than describe processing only one time